File: src/main/scala/org/evertones/persistence/modules/product/ComponentRepository.scala
package org.evertones.persistence.modules.product

import org.evertones.model.modules.product.Component
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.querydsl.QueryDslPredicateExecutor

trait ComponentRepository extends JpaRepository[Component, Integer]
        with QueryDslPredicateExecutor[Component] {

}

File: src/main/java/org/evertones/persistence/modules/product/ComponentRepository.java
package org.evertones.persistence.modules.product;

import org.evertones.model.modules.product.Component;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.querydsl.QueryDslPredicateExecutor;

public interface ComponentRepository
        extends JpaRepository<Component, Integer>,
            QueryDslPredicateExecutor<Component> {
}
